llama_stack/cli/stack/run.py

```diff
@@ -167,6 +174,19 @@ def get_conda_prefix(env_name):
                 script,
                 image_name,
             ]
+        elif args.image_type == ImageType.venv.value:
+            current_venv = os.environ.get("VIRTUAL_ENV")
+            script = importlib.resources.files("llama_stack") / "distribution/start_venv.sh"
+            run_args = [
+                script,
+                current_venv,
+            ]
+        else:
+            cprint(
+                "Invalid image type, please specify --image-type.",
+                color="red",
+            )
+            return
```

Comment:
This code is unreachable though since the parser checks the value against the possible choices.

Reply:
true! I will just flip the `venv` check to an else and drop this one